src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -52,21 +52,16 @@ public FeedbackQuestionAttributes(FeedbackQuestion fq) {
         this.showGiverNameTo = fq.getShowGiverNameTo();
         this.showRecipientNameTo = fq.getShowRecipientNameTo();
     }
-    
-    public FeedbackQuestionAttributes(
-            String feedbackSessionName,
-            String courseId,
-            String creatorEmail,
-            Text questionMetaData,
-            int questionNumber,
-            FeedbackQuestionType questionType,
-            FeedbackParticipantType giverType,
-            FeedbackParticipantType recipientType,
-            int numberOfEntitiesToGiveFeedbackTo,
-            List<FeedbackParticipantType> showResponsesTo,
-            List<FeedbackParticipantType> showGiverNameTo,
-            List<FeedbackParticipantType> showRecipientNameTo) {
-        
+
+    public FeedbackQuestionAttributes(String feedbackSessionName, String courseId, String creatorEmail,
+                                      Text questionMetaData, int questionNumber,
+                                      FeedbackQuestionType questionType,
+                                      FeedbackParticipantType giverType,
+                                      FeedbackParticipantType recipientType,
+                                      int numberOfEntitiesToGiveFeedbackTo,
+                                      List<FeedbackParticipantType> showResponsesTo,
+                                      List<FeedbackParticipantType> showGiverNameTo,
+                                      List<FeedbackParticipantType> showRecipientNameTo) {
```

Comment:
It is mostly a single-column stack, so for consistency sake, can make entirely a single-column. Alternatively, reduce the indentation (use 8 spaces?) and pack more parameters into each line.